src/Microsoft.Azure.WebJobs.Host/Indexers/AppSettingOrEnvironmentNameResolver.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Indexers
+{
+    /// <summary>
+    /// An implementation of <see cref="INameResolver"/> that resolves tokens by looking first
+    /// in App Settings and then in environment variables.
+    /// </summary>
+    internal class AppSettingOrEnvironmentNameResolver : INameResolver
```

Comment:
Sure, that makes sense. I was trying to be descriptive, but the comments should take care of that. Will also make public as you mentioned elsewhere.

---
In reply to: [69617436](https://github.com/Azure/azure-webjobs-sdk/pull/760#discussion_r69617436) [](ancestors = 69617436)